docs/fsharp-cheatsheet.md

```diff
@@ -1139,7 +1139,26 @@ open type System.Text.RegularExpressions.Regex  // type
 let isHttp url = IsMatch("^https?:", url)  // Regex.IsMatch directly accessible
 ```
 
-Available to `module` declarations only, is the `AutoOpen` attribute, which alleviates the need for an `open`.
+### RequireQualifiedAccess Attribute
+
+This attribute can be used on modules, records, and discriminated unions to avoid namespace collisions and unexpected "shadowing". It can also make your code more explicit and readable.
+
+```fsharp
+[<RequireQualifiedAccess>]
+module Math =
```

Comment:
maybe a different name e.g. Calcs to not clash with System.Math ?

Reply:
Good point. Done.